receiver/prometheusreceiver/internal/otlp_metricsbuilder_test.go

```diff
@@ -1036,6 +1038,7 @@ func Test_OTLPMetricBuilder_histogram(t *testing.T) {
 }
 
 func Test_OTLPMetricBuilder_summary(t *testing.T) {
+	// fix
```

Comment:
What needs to be fixed here?

Reply:
sorry it was mistakenly added. had just removed it.